frontend/simple/views/components/GroupSettings.vue

```diff
@@ -1,25 +1,23 @@
 <template>
-  <section>
-    <!-- Group Settings -->
-    <h3 class="title is-3"><i18n>Current Settings</i18n></h3>
+  <dashboard-section title="Current Settings">
```

Comment:
same goes here and all the rest of the sections: I suggest moving the `dashboard-section` wrapper out to GroupDashboard